### PR TITLE
Added the Filter by Users to Vacations Page

### PR DESCRIPTION
**Pull request by tbyrd2237**

### CODE REVIEW

File: backend/index.ts [hunk 1]
```diff
@@ -22,8 +22,10 @@ app.use(
   })
 );
 
-app.get("/api/vacations", vacationQueries.getVacations);
-app.post("/api/vacations/", vacationQueries.createVacation);
+// app.get("/api/vacations", vacationQueries.getVacations);
+app.post("/api/vacations", vacationQueries.createVacation);
```
Comment (lesleytomosada) [inline backend/index.ts:26]: i think this line can be deleted

File: backend/src/middleware/authenticate.js [hunk 2]
```diff
@@ -1,17 +1,42 @@
...
+
+//     const user = jwt.verify(cookie, "supersecret");
+
+//     req.user = user;
+//     next();
+//   } catch (error) {
+//     error.status = 401;
+//     next(error);
+//   }
+// };
```
Comment (lesleytomosada) [inline backend/src/middleware/authenticate.js:18]: don't forget to delete all the commented stuff if you dont need it

File: backend/src/controller/vacation.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+const { Router } = require('express');
+//Import express
+const authenticate = require('../middleware/authenticate');
+
+// console.log(authenticate);
```
Comment (aidanliddiard) [inline backend/src/controller/vacation.js:5]: when you are done you might want to delete this console.log

**Review by aidanliddiard — APPROVED** [review]
Yay it works!!!!! Looks good!

**Review by lesleytomosada — APPROVED** [review]
after we get the unsplash working reminder to delete the paris.jpg, looks great otherwise!

**Pull request merged**